python/lsst/pipe/analysis/utils.py

```diff
@@ -1231,7 +1231,7 @@ def addRotPoint(catalog, width, height, nQuarter, prefix=""):
 
 
 def makeBadArray(catalog, flagList=[], onlyReadStars=False, patchInnerOnly=True, tractInnerOnly=False,
-                 excludeIsolatedModel=False, excludeIsolatedNotModel=True, excludeSkipped=True):
+                 useScarletModelSource=False, excludeSkipped=True):
```

Comment:
I recommend calling this `useScarletIsolated`, since the blended sources will be scarlet models even if the isolated sources aren't. I also recommend changing the docstring to make this point clear.